config/test.json

```diff
@@ -55,12 +55,14 @@
     },
     {
       "name": "ANT",
-      "address": "0x960b236a07cf122663c4303350609a66a7b288c0",
+      "address": "0xe78A0F7E598Cc8b0Bb87894B0F60dD2a88d6a8Ab",
```

Comment:
I changed the address of  `ANT` in test.json to work with seed data tokenAddress that recently added,
it's no big deal